src/zoid/buttons/component.jsx

```diff
@@ -471,6 +472,28 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                 type:       'boolean',
                 queryParam: true,
                 required:   false
+            },
+
+            supportsPopups: {
+                type:       'boolean',
+                value:      () => userAgentSupportsPopups(),
+                queryParam: true
+            },
+
+            supportedNativeBrowser: {
+                type:       'boolean',
+                value:      () => {
+                    if (isIos() && isSafari()) {
```

Comment:
Mind re-adding `!supportsPopups() -> return false` here? Just for safety.

Reply:
Sure thing. I just pushed up this change. 

Thanks for reviewing again! 😄